The-Coding-Sharks-api/src/main/java/com/example/The_Coding_Sharks_demo/models/DTO/LoginFormDTO.java

```diff
@@ -0,0 +1,77 @@
+package com.example.The_Coding_Sharks_demo.models.DTO;
+
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+
+import static com.example.The_Coding_Sharks_demo.controllers.AuthenticationController.setUserInSession;
+
+public class LoginFormDTO {
+
+    @NotNull
+    @NotBlank
+    @Size(min = 3, max = 20, message = "Invalid username. Must be between 3 and 20 characters.")
+    private String username;
+
+    @NotNull
+    @NotBlank
+    @Size(min = 5, max = 30, message = "Invalid password. Must be between 5 and 30 characters.")
+    private String password;
+
+    public String getUsername() {
+        return username;
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
+        this.password = password;
+    }
+
+
```

Comment:
For clarity, could we remove this commented code?

Reply:
absolutely. Wanted to make sure I remembered to tell Shelly about this. thanks for pointing this out. I will remove it.